runtime/parachains/src/paras/mod.rs

```diff
@@ -1712,6 +1720,10 @@ impl<T: Config> Pallet<T> {
 			}
 		});
 
+		if <T as Config>::UmpQueueTracker::message_count(id) != 0 {
+			return Err(Error::<T>::CannotOffboard.into())
```

Comment:
Is is fine to just let `schedule_para_cleanup` error? Could the para-chain not lengthen its slot by keep sending UMPs?

Reply:
AFAIK "offboarding" is about actually removing the para from existance, not just cessation of scheduling. So you could only work against yourself here.